src/VisualStudio/Core/Def/Implementation/Venus/VenusCommandFilter.cs

```diff
@@ -24,18 +24,17 @@ internal class VenusCommandFilter<TPackage, TLanguageService> : AbstractVsTextVi
         public VenusCommandFilter(
             TLanguageService languageService,
             IWpfTextView wpfTextView,
-            ICommandHandlerServiceFactory commandHandlerServiceFactory,
```

Comment:
this is going to break TypeScript I think: https://devdiv.visualstudio.com/DevDiv/_git/TypeScript-VS?path=%2FVS%2FLanguageService%2FTypeScriptVisualStudio%2FVsPackage%2FScriptBlockHandler.cs&version=GBmaster&line=27&lineStyle=plain&lineEnd=28&lineStartColumn=1&lineEndColumn=1